src/domain/index.ts

```diff
@@ -250,12 +253,17 @@ export function handleExecuteProposal(event: ExecuteProposal): void {
     }
 }
 
-export function handleStateChange(event: StateChange): void {
-  if (isProposalValid(event.params._proposalId.toHex())) {
-      updateProposalState(event.params._proposalId, event.params._proposalState, event.address);
-      if ((event.params._proposalState === 1) ||
-          (event.params._proposalState === 2)) {
-          insertGPRewards(event.params._proposalId, event.block.timestamp, event.address, event.params._proposalState);
+export function handleStateChange(
```

Comment:
this change is not needed .
see https://github.com/daostack/subgraph/pull/170/files#r272109104